pkg/controller/account/account_controller.go

```diff
@@ -369,48 +369,13 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{}, err
 		}
 
-		// Create SRE IAM user and return the credentials
-		SREIAMUserSecret, err := r.BuildIAMUser(reqLogger, awsAssumedRoleClient, currentAcctInstance, iamUserSRE, request.Namespace)
+		// Create SRE IAM user, we don't care about the credentials because they're saved inside of the build func
+		_, err = r.BuildIAMUser(reqLogger, awsAssumedRoleClient, currentAcctInstance, iamUserSRE, request.Namespace)
 		if err != nil {
 			r.setStatusFailed(reqLogger, currentAcctInstance, fmt.Sprintf("Failed to build IAM SRE user: %s", iamUserSRE))
 			return reconcile.Result{}, err
 		}
 
-		// Intermittently our secret wont be ready before the next call, lets ensure it exists
-		for i := 0; i < 10; i++ {
-			secret := &corev1.Secret{}
-			err := r.Client.Get(context.TODO(), types.NamespacedName{Name: *SREIAMUserSecret, Namespace: request.Namespace}, secret)
```

Comment:
@iamkirkbater you need to remove `types` from the imports its not used in this file